common/scripts/README.md

```diff
@@ -0,0 +1,91 @@
+# Install Scripts
```

Comment:
We really do not need README files checked in describing an issue that has been fixed; once the issue is fixed, its not really relevant anymore